loopchain/peer/peer_outer_service.py

```diff
@@ -181,22 +149,16 @@ def __get_status_peer_type_data(self, channel: str):
         status['leader'] = status_cache['leader']
         return status
 
-    def __get_status_from_cache(self, channel: str):
-        if channel in self.peer_service.status_cache:
-            if channel in self.__status_cache_update_time:
-                update_time = self.__status_cache_update_time[channel]
-                if utils.datetime_diff_in_mins(update_time) > conf.STATUS_CACHE_LAST_UPDATE_IN_MINUTES:
-                    return None
-            status_data = self.peer_service.status_cache[channel]
-        else:
-            channel_stub = StubCollection().channel_stubs[channel]
-            status_data = asyncio.run_coroutine_threadsafe(
-                channel_stub.async_task().get_status(),
-                self.peer_service.inner_service.loop
-            ).result()
-            self.peer_service.status_cache[channel] = status_data
+    @lru_cache(maxsize=4, valued_returns_only=True)
```

Comment:
It stores status cache, but the only latest second signature is valid. (unless loopchain use `getStatus` to get data passed.)

Is there any reason for you to set `maxsize` as 4?
Every return value has large `dict`, contains many key-and-values.

I think 1 is enough.

Reply:
Basically, the GetStatus return value is used for monitoring. (Except block sync). So 1 second delay is no problem in its purpose. And, I agree your opinion about maxsize of cache.